src/services/experimental-features.test.js

```diff
@@ -0,0 +1,65 @@
+import { ExperimentalFeatures } from './experimental-features';
+import { FakeLocalStorage } from './local-storage-double';
+
+describe('experimental features', () => {
+  it('should validate if a feature is enabled', () => {
+    // Arrange
+    const experimentalFeaturesData = {
+      feature1: true,
+    };
+    const storage = new FakeLocalStorage();
+    storage.setItem('dopplerExpermiental', JSON.stringify(experimentalFeaturesData));
+    const experimentalFeatures = new ExperimentalFeatures(storage);
+    // Act
+    const enabled = !!experimentalFeatures.getFeature('feature1');
+
+    // Assert
+    expect(enabled).toBe(true);
+  });
+  it('should validate when a feature is not enabled', () => {
+    // Arrange
+    const experimentalFeaturesData = {
+      feature1: true,
+    };
+    const storage = new FakeLocalStorage();
+    storage.setItem('dopplerExpermiental', JSON.stringify(experimentalFeaturesData));
+    const experimentalFeatures = new ExperimentalFeatures(storage);
+    // Act
+    const enabled = !!experimentalFeatures.getFeature('feature4');
+
+    // Assert
+    expect(enabled).toBe(false);
+  });
+  it('should get a value to use in experimental feature', () => {
+    // Arrange
+    const experimentalFeaturesData = {
+      feature3: {
+        apikey: 'myApiKey',
+      },
+    };
+    const storage = new FakeLocalStorage();
+    storage.setItem('dopplerExpermiental', JSON.stringify(experimentalFeaturesData));
+    const experimentalFeatures = new ExperimentalFeatures(storage);
+    // Act
+    const value = experimentalFeatures.getFeature('feature3').apikey;
+
+    // Assert
+    expect(value).toBe('myApiKey');
+  });
+  it('should return null when feature value does not exist', () => {
```

Comment:
What do you think about adding some tests for _falsy_ feature values?

For example:

```js
// Arrange
const experimentalFeaturesData = {
  feature1: false // and also 0, '', null, undefined
};

// . . .

// Act
const value = experimentalFeatures.getFeature('feature1');

// Assert
expect(value).toBeNull;
```

Reply:
This [Jest functionality](https://jestjs.io/docs/en/api.html#2-describeeach-table-name-fn-timeout) looks great to test the same with different input values:

```js
describe.each`
  featureValue
  ${null}
  ${0}
  ${false}
  ${''}
`('should return null when feature value is falsy', ({featureValue}) => {
  test(`(featureValue is ${featureValue})`, () => {
    // Arrange
    const experimentalFeaturesData = {
      feature1: featureValue
    };

    // . . .

    // Act
    const value = experimentalFeatures.getFeature('feature1');

    // Assert
    expect(value).toBeNull;
  });
});

```

Reply:
This is very nice and work wonders. 😄